File: Morning-snacks/GreatestCommonDivisor.js
let firstNumber = prompt("Enter first number: ");
let secondNumber = prompt("Enter second number: ");

let gcd;

	for(count = 1; count <= firstNumber && count <= secondNumber; count++){

		if(firstNumber % count == 0 && secondNumber % count == 0){

		gcd = count;

		}


			

	}

	System.out.println("GCD is " + gcd);

File: Morning-snacks/greatest_common_divisor.py
first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))

gcd = 0

for count in range(1, max(first_number, second_number) + 1):

	if first_number % count == 0 and second_number % count == 0:

		gcd = count

print(gcd)